packaging/windows/BUILD.md

```diff
@@ -56,8 +56,8 @@ How to make a darktable Windows installer (x64 only):
         ```
     * If you have to set them manually you can do so by setting the variables in your `~/.bash_profile`. Example:
         ```bash
-        export CAMLIBS="/mingw64/lib/libgphoto2/2.5.30/"
-        export IOLIBS="/mingw64/lib/libgphoto2_port/0.12.1/"
+        export CAMLIBS="/UCRT64/lib/libgphoto2/2.5.30/"
+        export IOLIBS="/UCRT64/lib/libgphoto2_port/0.12.1/"
```

Comment:
Ah, some these replacements should have been case sensitive, sorry for missing that in the first review!